ecl/hql/hqlmanifest.cpp

```diff
@@ -198,6 +198,31 @@ void ResourceManifest::loadResource(const char *filepath, MemoryBuffer &content)
     read(fio, 0, (size32_t) f->size(), content);
 }
 
+void queryJfrog(const char *localpath, IPropertyTree &item)
+{
+    const char *repo = item.queryProp("@repository");
+    if(repo)
+    {
+        StringBuffer filename(localpath);
+        getFileNameOnly(filename, false);
+        StringBuffer filepath;
+        StringBuffer machine;
+        if (repo[0] == '/' && repo[1] == '/')
```

Comment:
What are the security implications of this change?  As far as I can see it would allow me to overwrite any local file with something stored in a jfrog repository.

Reply:
Yes, jfrog would overwrite the file with a resource from artifactory. I have changed it to write to a temporary directory.